Add additional payload.Hash examples (#2145)
ava-labs/avalanchego: vms/platformvm/warp/payload/README.md

- `codecID` is the codec version used to serialize the payload and is hardcoded to `0x0000`
- `typeID` is the payload type identifier and is `0x00000000` for `Hash`

> - `hash` is a hash from the `sourceChainID`. The format of the expected preimage is chain specific. Some examples for valid uses of the hash are:
>   - Hash is the root of a merkle tree
>   - Hash of a block that was accepted on the source chain
>   - Hash of a transaction that was included on the source chain

Reviewer: The `Hash` in the bullet points seems a little repetitive, maybe something like this would be better?

```suggestion
- `hash` is a hash from the `sourceChainID`. The format of the expected preimage is chain specific. Some examples for valid hash values are:
  - root of a merkle tree
  - accepted block hash on the source chain
  - accepted transaction hash on the source chain
```
